astropy/time/core.py

```diff
@@ -124,6 +124,16 @@ class Time(object):
     formats (e.g. JD) where very high precision (better than 64-bit precision)
     is required.
 
+    The allowed values for ``format`` and ``scale`` can be listed with::
+
+      >>> list(Time.FORMATS)
+      ['jd', 'mjd', 'decimalyear', 'unix', 'cxcsec', 'gps', 'plot_date', 'astropy_time',
+       'datetime', 'iso', 'isot', 'yday', 'fits', 'byear', 'jyear', 'byear_str',
+       'jyear_str']
+
+      >>> list(Time.SCALES)
+      ['tai', 'tcb', 'tcg', 'tdb', 'tt', 'ut1', 'utc']
```

Comment:
Same with @cdeil.  By the way, although not immediately useful to everyone, Python 3.5 is getting a C implementation of OrderedDict in the CPython core, so it will be that much better.  It still has more overhead than an unordered dict of course, but for many applications that doesn't matter.